docs/reference/mount.md

```diff
@@ -112,6 +112,19 @@ Flags
 --presign: Use presign for downloading.
 ```
 
+### Partial Reads
+
+{: .d-inline-block }
+experimental
+{: .label .label-red }
+
+Everest can read only a subsets of files on demand from lakeFS.
+This is useful for large files that are not needed at once, 
+or for files that are too large to fit in memory.
```

Comment:
```suggestion
This can be useful for streaming workloads or for applications handling file formats such as Parquet, m4a, zip, tar that do not need to read the entire file.
```